vars/buildPlugin.groovy

```diff
@@ -103,22 +99,10 @@ def call(Map params = [:]) {
                                 if (javaLevel) {
                                     mavenOptions += "-Djava.level=${javaLevel}"
                                 }
-                                if (params?.findbugs?.run || params?.findbugs?.archive) {
-                                    mavenOptions += "-Dfindbugs.failOnError=false"
-                                }
                                 if (skipTests) {
-                                    mavenOptions += "-DskipTests"
-                                }
-                                if (params?.checkstyle?.run || params?.checkstyle?.archive) {
-                                    mavenOptions += "-Dcheckstyle.failOnViolation=false -Dcheckstyle.failsOnError=false"
+                                    mavenOptions += "-DskipTests -DskipITs"
```

Comment:
`-DskipITs` is unrelated to the change subject. YAGNI?

Reply:
Yes, this flag is from failsafe plugin, I'll remove it.